response_operations_ui/templates/reporting-unit.html

```diff
@@ -152,16 +152,20 @@ <h4 class="neptune">Respondents</h4>
               </dl>
             </td>
             <td name="tbl-respondent-status" class="table--cell">
-              {{ respondent.status }}
+              {{ respondent.status.title() }}
             </td>
+
             <td name="tbl-enrolment-status" class="table--cell">
-              {{ respondent.enrolmentStatus }}
-                {% if respondent.enrolmentStatus|upper == "PENDING" %}
```

Comment:
I don't think you meant to delete this, we should still have the resend verification email link

Reply:
Added it back in 👍